content/rdi/_index.md

```diff
@@ -40,6 +42,7 @@ Redis Data Integration supports the following database sources using [Debezium S
 | SQL Server         | 2017, 2019             |
 | Cassandra          | >= 3.0                 |
 | Datastax Cassandra | >= 6.8.0               |
+| MongoDB            | 4.2, 4.4, 5.0, 6.0  driver: 4.7 |
```

Comment:
Not sure if the driver related to all those versions or just to the last one. Maybe add another column named Notes/Comments with this piece of information.

Reply:
it is taken from debezium docs

Reply:
done